diff --git a/source/audioio/inc/AOCoreAudioMacOS.h b/source/audioio/inc/AOCoreAudioMacOS.h
index 47ec680..8b71389 100644
--- a/source/audioio/inc/AOCoreAudioMacOS.h
+++ b/source/audioio/inc/AOCoreAudioMacOS.h
@@ -26,6 +26,15 @@ typedef UInt32 AudioFormatID;
 typedef UInt32 AudioFormatFlags;
 #endif

+#if MAC_OS_X_VERSION_MIN_REQUIRED < 1060
+typedef Component AudioComponent;
+typedef ComponentInstance AudioComponentInstance;
+typedef ComponentDescription AudioComponentDescription;
+#define AudioComponentFindNext FindNextComponent
+#define AudioComponentInstanceNew OpenAComponent
+#define AudioComponentInstanceDispose CloseComponent
+#endif
+
 //-------------------------------------------------------------------------------------------
 namespace omega
 {
diff --git a/source/audioio/src/AOCoreAudioMacOS.mm b/source/audioio/src/AOCoreAudioMacOS.mm
index aa75eda..7293af0 100644
--- a/source/audioio/src/AOCoreAudioMacOS.mm
+++ b/source/audioio/src/AOCoreAudioMacOS.mm
@@ -21,6 +21,20 @@
 #define kAudioObjectPropertyElementMain kAudioObjectPropertyElementMaster
 #endif

+#if MAC_OS_X_VERSION_MIN_REQUIRED < 1060
+typedef Component AudioComponent;
+typedef ComponentInstance AudioComponentInstance;
+typedef ComponentDescription AudioComponentDescription;
+#define AudioComponentFindNext FindNextComponent
+#define AudioComponentInstanceNew OpenAComponent
+#define AudioComponentInstanceDispose CloseComponent
+#define kAudioFormatAES3 'aes3'
+#define kAudioFormatAudible 'AUDB'
+#define kAudioFormatDVIIntelIMA 0x6D730011
+#define kAudioFormatMicrosoftGSM 0x6D730031
+#define kAudioFormatiLBC 'ilbc'
+#endif
+
 //-------------------------------------------------------------------------------------------
 namespace omega
 {
diff --git a/source/audioio/src/AOQueryCoreAudio.mm b/source/audioio/src/AOQueryCoreAudio.mm
index fec03ce..4296de2 100644
--- a/source/audioio/src/AOQueryCoreAudio.mm
+++ b/source/audioio/src/AOQueryCoreAudio.mm
@@ -14,6 +14,15 @@ typedef ushort omegashort;
 #define kAudioObjectPropertyElementMain kAudioObjectPropertyElementMaster
 #endif

+#if MAC_OS_X_VERSION_MIN_REQUIRED < 1060
+typedef Component AudioComponent;
+typedef ComponentInstance AudioComponentInstance;
+typedef ComponentDescription AudioComponentDescription;
+#define AudioComponentFindNext FindNextComponent
+#define AudioComponentInstanceNew OpenAComponent
+#define AudioComponentInstanceDispose CloseComponent
+#endif
+
 //-------------------------------------------------------------------------------------------
 namespace omega
 {
